.github/workflows/on-push-tests.yml

```diff
@@ -0,0 +1,78 @@
+name: Run tests
+on: push
+
+jobs:
+  prepare_dependencies:
+    name: Prepare dependencies
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12"
+
+      - name: Get cached dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            node_modules 
+            packages/vue/node_modules
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install dependencies
+        run: yarn --frozen-lockfile && yarn add @vue/cli -W
```

Comment:
Is there a reason for not including `@vue/cli` in package.json` (probably as `devDependency`)?

Reply:
I think there is no reason for that, I'll add it.